src/pages/TransactionsPage/TransactionsForm/TransactionsForm.js

```diff
@@ -0,0 +1,122 @@
+import { StyledTransactionForm } from "./TransactionsForm.styled";
+import { DatePicker } from "../../../components/form/DatePicker/DatePicker";
+import { Input } from "../../../components/form/Input/Input";
+import { RadioInput } from "../../../components/form/RadioInput/RadioInput";
+import { DropdownInput } from "../../../components/form/DropdownInput/DropdownInput";
+import { Button } from "../../../components/Button/Button";
+import { useInputState } from "../../../services/hooks/useInputState";
+import { useState, useEffect } from "react";
+
+export const TransactionForm = (closeModal) => {
+  const [title, setTitle, resetTitle] = useInputState("");
+  const [description, setDescription, resetDescription] = useInputState("");
+  const [amount, setAmount, resetAmount] = useInputState(0);
+  const [type, setType, resetType] = useInputState("income");
+  const [date, setDate, resetDate] = useInputState("");
+  const [category, setCategory] = useState("select");
+  const [paymentType, setPaymentType, resetPaymentType] = useInputState("cash");
+
+  const [formValues, setFormValues] = useState({
```

Comment:
do we need this useState if we have each state from either useState or useInputState which is 2 lines above? especially when `formValues` is not used